pynitrokey/cli/fido2.py

```diff
@@ -558,7 +583,7 @@ def set_pin(serial):
 
 
 @click.command()
-# @click.option("--pin", help="PIN for to access key", default=None)
+@click.option("--pin", help="PIN for device access, use '-' for a prompt", default=None)
 @click.option(
     "-s",
```

Comment:
I understand `-` here is only for a quick adaptation of the current implementation.
However, in general it should be implemented in a way, that PIN is requested once the reply received contains error code saying it is required, and then the whole operation should be repeated afterwards. Being rejected with this error does not cost anything (specifically PIN attempts counter), hence it would be best to query device first. No need to overcomplicate this.

Reply:
see discussion